spec/01-lexical-syntax.md

```diff
@@ -11,7 +11,7 @@ Scala programs are written using the Unicode Basic Multilingual Plane
 presently supported.  This chapter defines the two modes of Scala's
 lexical syntax, the Scala mode and the _XML mode_. If not
 otherwise mentioned, the following descriptions of Scala tokens refer
-to _Scala mode_, and literal characters ‘c’ refer to the ASCII fragment
+to _Scala mode_, and literal characters ‘c’ refers to the ASCII fragment
```

Comment:
`characters` is the subject here. Perhaps `'c'` should be offset to emphasize that it's not just the letter 'c'.